packages/shared/src/modules/users/libs/types/user.type.ts

```diff
@@ -0,0 +1,6 @@
+type User = {
```

Comment:
```suggestion
type UserAuthResponseDto = {
```
Don't forget about the file name